electric_emission_cost/costs.py

```diff
@@ -365,12 +377,26 @@ def get_charge_df(
         for key, value in charge_dict.items()
         if any(k in key for k in ["electric_customer", "gas_customer"])
     }
+    if scale_charges:
```

Comment:
I would name this `scale_fixed_charges` because `demand_scale_factor` is already an argument to `calculate_cost`

Reply:
Thinking more about this, I see the benefit of scaling the charges outside of `calculate_cost`, so I'd say go ahead and keep the scaling of demand charges but break it into two separate flags: `scale_fixed_charges` and `scale_demand_charges`

Reply:
And then if by default `scale_demand_charges=False` there shouldn't be any issues with users accidentally scaling them twice